src/js/components/Share/ShareModal.jsx

```diff
@@ -27,6 +27,7 @@ import MessageCard from '../Widgets/MessageCard';
 import { CopyLink, getKindOfShareFromURL, saveActionShareAnalytics, ShareFacebook, SharePreviewFriends, shareStyles, ShareTwitter, ShareWeVoteFriends } from './shareButtonCommon'; // cordovaSocialSharingByEmail
 import { generateShareLinks } from './ShareModalText';
 import ShareModalTitleArea from './ShareModalTitleArea';
+import BallotStore from "../../stores/BallotStore";
```

Comment:
Thank you @Anvesh-ch, I'm approving this pull request. 👍  Don't forget to run `npm run lint` to catch ESLINT warnings like using single quotes instead of double quotes for imports.